### ISSUE TITLE
troubles in data_preparation

### ISSUE THREAD

**Issue by kfmn** [BUTSpeechFIT/EEND_dataprep#7]
Hi,

Thank you for the recipes provided. 
But I have found some obstacles when preparing LibriSpeech-based dataset.
1. There is no information in v2/LibriSpeech/README.md about setting KALDI_ROOT in path.sh and about neseccity of running prepareKaldidata_LibriSpeech.sh prior to running generate_data.sh
2. After preparing Kaldi data with prepareKaldidata_LibriSpeech.sh the Kaldidatadir/alltrain_persession subfolder contains data with utterance-ids prefixed with "lbi-" while generate_data.sh doesn't use this prefix, which results in that conv2wav.py expects uttIDs without prefix. This causes generate_data.sh not to generate wavs due to KeyErrors in conv2wav.py, line 111.  I had to manually remove prefix from kaldi uttids to overcome this.
3. generate_data.sh should source path.sh in its preamble ( . ./path.sh ). Otherwise conv2wav.py's call to wav-reverberate fails since bash cannot locate it.

Hope these observations will be useful for some fixing of code
Best regards,
Maxim

**Comment by fnlandini** [comment]
Dear Maxim,
Thank you for the feedback. I have just updated the READMEs of the v2 recipes to be similar to v1. That covers point 1. I also added the call to path.sh you mention in point 3.
Regarding point 2, somebody had already raised this in an [issue](https://github.com/BUTSpeechFIT/EEND_dataprep/issues/5#issuecomment-1767796795) and I have just realized that I was using an older version of the data_prep.sh script from Kaldi for LibriSpeech which did not include the "lbi-" prefixes. I have updated `prepareKaldidata_LibriSpeech.sh` to remove those prefixes.

Thank you again and let me know if you find any problems.
Federico
